src/Orchestration/Adapter/K8sCLI.php

```
@@ -0,0 +1,854 @@
+
+        // In K8s, we add a network label to the pod
+        $labelValue = $this->sanitizeLabelValue($network);
+        $result = Console::execute($this->buildKubectlCmd().' label pod '.\escapeshellarg($container).' network='.\escapeshellarg($labelValue).' --overwrite', '', $output);
+
+        return $result === 0;
+    }
+
+    /**
+     * Disconnect a container from a network (Remove label from pod)
+     */
+    public function networkDisconnect(string $container, string $network, bool $force = false): bool
```

PHPMD (2.15.0)

Avoid unused parameters such as '$network'. (undefined)

(UnusedFormalParameter)